src/core/annotation.js

```diff
@@ -242,6 +254,7 @@ class Annotation {
   constructor(params) {
     const dict = params.dict;
 
+    this.defaultAppearanceData = params.defaultAppearanceData;
```

Comment:
I think this can be removed since the only place that uses `defaultAppearanceData` is in the `TextWidgetAnnotation` class and it fetches it from the `params`, not from this member variable.